outputs.tf

```
@@ -0,0 +1,27 @@
+output "detector" {
+  description = "GuardDuty filter"
+  value       = aws_guardduty_detector.this
+}
+output "filter" {
+  description = "GuardDuty filter"
+  value       = var.filter != null ? aws_guardduty_filter.this : null
```

Can simplify these outputs:

```suggestion
  value       = aws_guardduty_filter.this
```

---

Good catch.  I missed this when i updated the filter default from null to an empty list.